.gitignore

```diff
@@ -0,0 +1,11 @@
+/.bundle/
+/doc/
+/log/*.log
+/pkg/
+/tmp/
+/test/dummy/db/*.sqlite3
+/test/dummy/db/*.sqlite3-*
```

Comment:
Can we remove sqlite related code changes from here and add if any changes required for pg? @MohammedNazeer10